sim/core/statweight.go

```diff
@@ -134,10 +134,13 @@ func CalcStatWeight(swr proto.StatWeightsRequest, statsToWeigh []stats.Stat, ref
 			panic("Stat weights error: " + errorStr)
 		}
 		dpsMetrics := simResult.RaidMetrics.Parties[0].Players[0].Dps
-		tpsMetrics := simResult.RaidMetrics.Parties[0].Players[0].Dps
+		tpsMetrics := simResult.RaidMetrics.Parties[0].Players[0].Threat
 		dtpsMetrics := simResult.RaidMetrics.Parties[0].Players[0].Dtps
 		dpsDiff := (dpsMetrics.Avg - baselineDpsMetrics.Avg) / value
 		tpsDiff := (tpsMetrics.Avg - baselineTpsMetrics.Avg) / value
+
+		// Im unsure this is a good way to define it, a negative result here is actually the
+		// good outcome, which is contrary to dps/tps.
```

Comment:
This comment is a bit confusing, I don't think this calculation favors positive or negative values?

Reply:
The idea is that while a dpsDiff > 0 is a thing that should result in a higher weight, the opposite is the case for dtps, as dtpsDiff < 0 is what should result in a higher weight.